include/boost/compute/kernel.hpp

```diff
@@ -371,6 +376,14 @@ class kernel
         detail::set_kernel_arg<T>()(*this, index, value);
     }
 
+    #ifdef BOOST_COMPUTE_USE_CPP11
```

Comment:
We should be able to replace that with `#ifndef BOOST_NO_CXX11_NULLPTR`.

Reply:
added in new commit